### PR TITLE
solution to cli issue - TypeError: Cannot read properties of undefined (reading 'fsPath') #895

### PR DESCRIPTION
**Pull request by gauravsaini04**
ref: #895 

**Description**
* The pr is raised as a solution to the issue `TypeError: Cannot read properties of undefined (reading 'fsPath')` #895.

### CODE REVIEW

**Review by chrmarti — CHANGES_REQUESTED** [review]
Good catch, left a comment for a possible improvement.

File: src/spec-node/featuresCLI/resolveDependencies.ts [hunk 1]
```diff
@@ -84,7 +84,9 @@ async function featuresResolveDependencies({
 		env: process.env,
 	};
 
-	const { lockfile } = await readLockfile(config);
+	let configObj: DevContainerConfig | string = config.hasOwnProperty('configFilePath') && config.configFilePath ? config : configPath;
```
[the remark this answers is not in the document]
Comment (gauravsaini04) [reply src/spec-node/featuresCLI/resolveDependencies.ts]: have implemented as you had suggested, need another review from you

**Review by chrmarti — CHANGES_REQUESTED** [review]
Looks good, one small ask, otherwise ready for merging. Thanks.

File: src/spec-node/featuresCLI/resolveDependencies.ts [hunk 2]
```diff
@@ -61,28 +68,29 @@ async function featuresResolveDependencies({
...
 
-	//  Parse dev container config
-	const config: DevContainerConfig = jsonc.parse(buffer.toString());
-	if (!config || !config.features) {
-		output.write(`No Features object in configuration '${configPath}'`, LogLevel.Error);
-		process.exit(1);
+	const cwd = workspaceFolder || process.cwd();
+	const cliHost = await getCLIHost(cwd, loadNativeModule, true);
+	const workspace = workspaceFromPath(cliHost.path, workspaceFolder);
+	const configFile: URI | undefined = configPath ? URI.file(path.resolve(process.cwd(), configPath)) : undefined;
```
[the remark this answers is not in the document]
Comment (gauravsaini04) [reply src/spec-node/featuresCLI/resolveDependencies.ts]: thanks ! made changes..

**Review by chrmarti — APPROVED** [review]
LGTM!

**Review by joshspicer — APPROVED** [review]

**Comment by gauravsaini04** [comment]
> LGTM!

Hi @chrmarti, as I don't have merging authorization, could you pls merge this

**Review by eljog — APPROVED** [review]

**Pull request merged**